certbot-nginx/certbot_nginx/configurator.py

```diff
@@ -257,6 +257,31 @@ def choose_vhost(self, target_name, raise_if_no_match=True):
 
         return vhost
 
+
+    def ipv6_info(self, port):
```

Comment:
would you mind moving this to `parser.py`? since it's used both here and in `tls_sni_01`.

Reply:
Wouldn't we be calling `parser` through `configurator` anyway from `tls_sni_01`?